cmd/harmony/main.go

```diff
@@ -138,11 +113,34 @@ func initSetup() {
 }
 
 func createGlobalConfig() *nodeconfig.ConfigType {
+	var err error
 	nodeConfig := nodeconfig.GetGlobalConfig()
 	nodeConfig.StakingPriKey = node.LoadStakingKeyFromFile(*stakingKeyFile, *accountIndex)
 
-	// Initialize leveldb if dbSupported.
+	nodeConfig.P2pPriKey, _, err = utils.LoadKeyFromFile(*keyFile)
+	if err != nil {
+		panic(err)
+	}
+
+	// Setup Bls keys
+	nodeConfig.BlsPriKey, nodeConfig.BlsPubKey = utils.GenKey(*ip, *port)
```

Comment:
A consistent naming convention would be helpful?  `P2pPriKey` refers to the key usage (P2P), whereas `BlsPriKey` refers to the key algorithm (BLS).  I'd prefer usage-based naming convention; how about `NodePriKey`?  (Plus, we don't want to change the variable name to something else in the future simply because we ditch BLS and use some other signature algorithm. 😄)

Reply:
let's discuss offline. NodePriKey is consusing to me. basically there are 3 set of keys with clear function.

- p2p keys
- account key (staking accounts)
- bls keys used to do multi-sig

NodePriKey does not imply anything with Bls

Reply:
Discussed offline.  FTR: `ConsensusPriKey`